service/version_bundles.go

```diff
@@ -115,5 +115,46 @@ func NewVersionBundles() []versionbundle.Bundle {
 			Version:      "1.0.0",
 			WIP:          false,
 		},
+		{
+			Changelogs: []versionbundle.Changelog{
+				{
+					Component:   "kubernetes",
+					Description: "Enable encryption at rest",
+					Kind:        "changed",
+				},
+			},
+			Components: []versionbundle.Component{
+				{
+					Name:    "calico",
+					Version: "2.6.2",
+				},
+				{
+					Name:    "docker",
+					Version: "1.12.6",
+				},
+				{
+					Name:    "etcd",
+					Version: "3.2.7",
+				},
+				{
+					Name:    "kubedns",
+					Version: "1.14.5",
+				},
+				{
+					Name:    "kubernetes",
+					Version: "1.8.4",
+				},
+				{
+					Name:    "nginx-ingress-controller",
+					Version: "0.9.0",
+				},
+			},
+			Dependencies: []versionbundle.Dependency{},
+			Deprecated:   false,
+			Name:         "kvm-operator",
+			Time:         time.Date(2017, time.December, 19, 10, 00, 0, 0, time.UTC),
+			Version:      "1.1.0",
+			WIP:          false,
```

Comment:
I thought about this. It would probably be good to keep this as being WIP first to have a couple of days where others can bring in other changes. E.g. @fgimenez has also something to contribute and we could just serve this under the same version bundle. No need to create two version bundles that quickly in a row.

Reply:
that was the plan, but WIP to `false` needed for testing

Reply:
Why you need WIP:false for testing? Can you set whatever in the custom object?

Reply:
He was also making sure the release aggregation worked.

Reply:
> E.g. @fgimenez has also something to contribute and we could just serve this under the same version bundle. No need to create two version bundles that quickly in a row.

#322 is already merged, I've created #326, targeted to this PR, to enable `deploymentv3` (tests failing because of missing merge/rebase to latest master).